pkg/logs/input/file/position.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package file
+
+import (
+	"io"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/auditor"
+)
+
+// Position returns the position from where logs should be collected.
+func Position(registry auditor.Registry, identifier string, tailFromBeginning bool) (int64, int, error) {
+	var offset int64
+	var whence int
+	var err error
+	value := registry.GetOffset(identifier)
+	if value != "" {
+		offset, err = strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			offset, whence = 0, io.SeekEnd
+		} else {
+			whence = io.SeekStart
```

Comment:
why setting the offset elsewhere but not here?

Reply:
ah nevermind

Reply:
I think to clarify, I'd put `whence = io.SeekStart` before the `offset, err = strconv.ParseInt(value, 10, 64)`
I'm not fan of `if err == nil set it` I find it harder to read